app/views/admin/kudos/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>
+  <strong>Title:</strong>
+  <%= @kudo.title %>
+</p>
+
+<p>
+  <strong>Content:</strong>
+  <%= @kudo.content %>
+</p>
+
```

Comment:
I think it would be good to display here also information about the kudo giver, and receiver

Reply:
I agree, It is now partially fixed ;)